os/drivers/os_api_test/application/test_app.c

```diff
@@ -0,0 +1,339 @@
+/****************************************************************************
```

Comment:
1. I don't know why this is `test_app`. File name should represent the operation. I think this is not.
2. This folder is `os_api_test`. If this is `test_app`, then it's not proper to put here.

Reply:
This is not the purpose of os internal function test and not app test. This driver is only valid when tc is enabled, but because of this, even tc is disabled, we can't disable this driver. I think it's not correct. We need to make a path for debug purpose. As far as I know, procfs treats some os information like cpu load, tcb info and so on. How about using procfs?
@jeongarmy Could you give your opinion?

Reply:
I think changing filename is enough because this is just called by testapp, not used by any commands like ps, cpuload and it is only enabled when CONFIG_DRIVERS_OS_API_TEST is enabled by makefile.

Reply:
@sunghan-chang Lets rename it then ?

Reply:
@thapav Sorry, I confused it. Is this function called by command `dat`? If it is right, I think we'd better to use procfs as he said.

Reply:
Updated.